### PR TITLE
Limit swerve motors

### PR DESCRIPTION
**Pull request by fizzlebert**

### CODE REVIEW

File: pyswervedrive/swervemodule.py [hunk 1]
```diff
@@ -87,6 +87,14 @@ def __init__(self, name: str,
 
         self.reset_encoder_delta()
 
+        self.steer_motor.configPeakCurrentLimit(40, timeoutMs=10)
+        self.steer_motor.configContinuousCurrentLimit(40, timeoutMs=10)
```
Comment (auscompgeek) [inline pyswervedrive/swervemodule.py]: Surely the peak and continuous current limits should be different?

File: pyswervedrive/swervemodule.py [hunk 2]
```diff
@@ -87,6 +87,14 @@ def __init__(self, name: str,
...
+        self.steer_motor.enableCurrentLimit(True)
+
+        self.drive_motor.configPeakCurrentLimit(40, timeoutMs=0)
```
Comment (auscompgeek) [inline pyswervedrive/swervemodule.py]: Do you really want this call to never time out?

**Review by auscompgeek — DISMISSED** [review]

File: pyswervedrive/swervemodule.py (hunk 1 above)
Comment (ArthurAllshire) [reply pyswervedrive/swervemodule.py]: Also, I'm not sure what the point of even setting these to 40 is, considering that (some of them at least) are on 30 amp breakers

Comment (auscompgeek) [reply pyswervedrive/swervemodule.py]: Yeah… I definitely recall discussion about plugging the steer motors into the 30 A breakers.

Comment (james-ward) [reply pyswervedrive/swervemodule.py]: The motor controllers will react faster than a thermal fuse.

Comment (auscompgeek) [reply pyswervedrive/swervemodule.py]: This is very true, but I wouldn't imagine you'd want to let the steer motors continuously draw 40 A on average either, especially if they're connected to a 30 A fuse (thermal or not).

**Review by ArthurAllshire — APPROVED** [review]

**Pull request merged**